keras_cv/layers/preprocessing/base_image_augmentation_layer.py

```diff
@@ -210,8 +211,28 @@ def augment_keypoints(self, keypoints, transformation, **kwargs):
         """
         raise NotImplementedError()
 
+    def augment_segmentation_mask(self, segmentation_mask, transformation, **kwargs):
+        """Augment a single image's segmentation mask during training.
+
+        Args:
+          segmentation_mask: 3D segmentation mask input tensor to the layer.
+            Forwarded from `layer.call()`.
+          transformation: The transformation object produced by
+            `get_random_transformation`. Used to coordinate the randomness
+            between image, label, and segmentation mask.
```

Comment:
"and segmentation mask" should be applied to all the other augment_x methods.

Reply:
Done -- also added keypoints, since that wasn't in the docstring.